tee-worker/scripts/litentry/release/prepare.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -euo pipefail
+
+# Edit the following variable(s):
+WORKER_DIR=/tmp/worker/
+
+RUNNING_MODE_CONFIG=$1/config.json
```

Comment:
`$1` is quite confusing: what should it be? We lack examples/explanations

Reply:
Refactored with fixed path: /opt/worker